client/state/jetpack-sync/reducer.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencis
+ */
+import { combineReducers } from 'redux';
+import pick from 'lodash/pick';
+import get from 'lodash/get';
```

Comment:
I think this can be rewritten as `import { pick, get } from 'lodash'; ` 
As per https://github.com/Automattic/wp-calypso/pull/6539/commits/b56e7722ca6858e1b4500458910c8da3f5f2913a